src/archanaNJun20/PrimeNumberOpration.java

```diff
@@ -0,0 +1,46 @@
+package archanaNJun20;
+
+import java.util.Scanner;
+
+public class PrimeNumberOpration {
+	void primeOperation(){
```

Comment:
make this method parameterized.

Reply:
The code is updated based on the given comments.
method names changed, from next time onwards  I will take care while writing.